includes/class-sensei-customizer.php

```diff
@@ -105,7 +105,7 @@ public function output_custom_settings() {
 
 		foreach ( $this->colors as $variable => $settings ) {
 			$value = get_option( $variable );
-			if ( $value ) {
+			if ( $value && $value !== $settings['default'] ) {
```

Comment:
I think this should avoid loading the customizer color if I reset to the default, right?

I reset it, but this is what I get. Something that I missed?

<img width="363" alt="Screen Shot 2022-01-28 at 18 24 48" src="https://user-images.githubusercontent.com/876340/151623521-ae46488c-c992-46ae-97df-5f2c1fe0e91c.png">

<img width="1341" alt="Screen Shot 2022-01-28 at 18 13 40" src="https://user-images.githubusercontent.com/876340/151623560-ae76e398-759f-470b-8bce-587005067b8c.png">

In the site editor and lesson editor it applied the colors, although in the site editor it didn't apply to some elements, like the course navigation and the top bar.

<img width="1366" alt="Screen Shot 2022-01-28 at 18 18 37" src="https://user-images.githubusercontent.com/876340/151623646-ad23ca38-15f7-42ef-a2ee-17261a5aa1ad.png">

<img width="1372" alt="Screen Shot 2022-01-28 at 18 13 26" src="https://user-images.githubusercontent.com/876340/151623660-2bb99d22-4e83-4e1c-ae27-3978855a25f2.png">

Reply:
Also putting this here for posterity, the way to customize the theme colors:
https://user-images.githubusercontent.com/176949/151625963-84dc5639-8046-41f2-b76b-632bcec63ce4.mov